src/pwncore/routes/ctf/start.py

```diff
@@ -144,13 +141,13 @@ async def stopall_docker_container(response: Response):
 
 @atomic()
 @router.post("/stop/{ctf_id}")
-async def stop_docker_container(ctf_id: int, response: Response):
+async def stop_docker_container(ctf_id: int, response: Response, jwt=require_jwt):
```

Comment:
This isn't what I mean't.
This will throw B008 and mypy will also complain

I meant doing this
```py
async def read_users(commons: Annotated[dict, Depends(common_parameters)]):
```

Reply:
B008 is still thrown, regardless of Annotated.

It is anyways solved by `require_jwt = Depends(get_jwt)` and using `require_jwt` as the default argument for jwt. (should've done this in the first place itself)